tests/unit/date/core.js

```diff
@@ -1,19 +1,21 @@
 define( [
 	"jquery",
-	"globalize",
+	"./helper",
 	"date"
-], function( $, Globalize ) {
+], function( $, testHelper ) {
 
 module( "date: core" );
 
+var attributes = testHelper.getAttributes( "en" );
+
 test( "Instantiation", function() {
 	expect( 2 );
-	ok( new $.date() instanceof $.date, "constructor function" );
-	ok( $.date() instanceof $.date, "instantiation without new" );
+	ok( new $.ui.calendarDate( null, attributes ) instanceof $.ui.calendarDate, "constructor function" );
```

Comment:
> Is it useful on its own? Is it intended to be used by others?

Yes. One could use it to ease up working with JS date objects without using calendar or datepicker at all. Pretty sure there are more fancy date helpers out there, but...